docs/creating-pdf-documents.md

```diff
@@ -87,6 +87,12 @@ See the official documentation for installation instructions for $\LaTeX$ at:
 
 Ensure that you download a full distribution with appropriate libraries installed.
 
+(install-tectonic)=
+
+#### Tectonic as a lightweight alternative
+As an alternative, you can also install [Tectonic](https://tectonic-typesetting.github.io/), "a modernized, complete, self-contained TeX/LaTeX engine, powered by XeTeX and TeXLive".<br>
+> "Tectonic automatically downloads support files so you don’t have to install a full LaTeX system in order to start using it. If you start using a new LaTeX package, Tectonic just pulls down the files it needs and continues processing." (excerpt from the website)
```

Comment:
Thanks for the docs!!

```suggestion
> Tectonic automatically downloads support files so you don’t have to install a full LaTeX system in order to start using it. If you start using a new LaTeX package, Tectonic just pulls down the files it needs and continues processing.
>
> -- [Tectonic Website](https://tectonic-typesetting.github.io/)
```

Reply:
Makes it look [like this](https://mystmd.org/sandbox?tab=demo&myst=PgAgAFQAZQBjAHQAbwBuAGkAYwAgAGEAdQB0AG8AbQBhAHQAaQBjAGEAbABsAHkAIABkAG8AdwBuAGwAbwBhAGQAcwAgAHMAdQBwAHAAbwByAHQAIABmAGkAbABlAHMAIABzAG8AIAB5AG8AdQAgAGQAbwBuABkgdAAgAGgAYQB2AGUAIAB0AG8AIABpAG4AcwB0AGEAbABsACAAYQAgAGYAdQBsAGwAIABMAGEAVABlAFgAIABzAHkAcwB0AGUAbQAgAGkAbgAgAG8AcgBkAGUAcgAgAHQAbwAgAHMAdABhAHIAdAAgAHUAcwBpAG4AZwAgAGkAdAAuACAASQBmACAAeQBvAHUAIABzAHQAYQByAHQAIAB1AHMAaQBuAGcAIABhACAAbgBlAHcAIABMAGEAVABlAFgAIABwAGEAYwBrAGEAZwBlACwAIABUAGUAYwB0AG8AbgBpAGMAIABqAHUAcwB0ACAAcAB1AGwAbABzACAAZABvAHcAbgAgAHQAaABlACAAZgBpAGwAZQBzACAAaQB0ACAAbgBlAGUAZABzACAAYQBuAGQAIABjAG8AbgB0AGkAbgB1AGUAcwAgAHAAcgBvAGMAZQBzAHMAaQBuAGcALgAKAD4ACgA%2BACAALQAtACAAWwBUAGUAYwB0AG8AbgBpAGMAIABXAGUAYgBzAGkAdABlAF0AKABoAHQAdABwAHMAOgAvAC8AdABlAGMAdABvAG4AaQBjAC0AdAB5AHAAZQBzAGUAdAB0AGkAbgBnAC4AZwBpAHQAaAB1AGIALgBpAG8ALwApAA%3D%3D)